Helper/Data.php

```diff
@@ -37,13 +42,14 @@ class Data extends BaseData
      * @param Initial $initialConfig
      */
     public function __construct(
-        Logger $logger,
-        Context $context,
+        Logger        $logger,
```

Comment:
This kinds of fixes don't use setup us, please check php-cs-fixer in MS project for reference. A better chance will be: don't annotations, use types for var, by example:

```php
- /**
- * @var Logger
- */
- protected $logger
+ protected Logger $logger
```